tools/matrix_analyzer.py

```diff
@@ -17,55 +17,56 @@
+            {'cre_type': 'vip', 'target_layer': '2/3', 'cortical_layer': '2/3'},
            # {'cre_type': 'rorb', 'target_layer': '4'},
-            {'cre_type': 'nr5a1', 'target_layer': '4'},
-            {'cre_type': 'pvalb', 'target_layer': '4'},
-            {'cre_type': 'sst', 'target_layer': '4'},
-            {'cre_type': 'vip', 'target_layer': '4'},
-            {'cre_type': 'sim1', 'target_layer': '5'},
-            {'cre_type': 'tlx3', 'target_layer': '5'},
-            {'cre_type': 'pvalb', 'target_layer': '5'},
-            {'cre_type': 'sst', 'target_layer': '5'},
-            {'cre_type': 'vip', 'target_layer': '5'},
-            {'cre_type': 'ntsr1', 'target_layer': '6'},
-            {'cre_type': 'pvalb', 'target_layer': '6'},
-            {'cre_type': 'sst', 'target_layer': '6'},
-            {'cre_type': 'vip', 'target_layer': '6'},
+            {'cre_type': 'nr5a1', 'target_layer': '4', 'cortical_layer': '4'},
+            {'cre_type': 'pvalb', 'target_layer': '4', 'cortical_layer': '4'},
+            {'cre_type': 'sst', 'target_layer': '4', 'cortical_layer': '4'},
+            {'cre_type': 'vip', 'target_layer': '4', 'cortical_layer': '4'},
+            {'cre_type': 'sim1', 'target_layer': '5', 'cortical_layer': '5'},
+            {'cre_type': 'tlx3', 'target_layer': '5', 'cortical_layer': '5'},
+            {'cre_type': 'pvalb', 'target_layer': '5', 'cortical_layer': '5'},
+            {'cre_type': 'sst', 'target_layer': '5', 'cortical_layer': '5'},
+            {'cre_type': 'vip', 'target_layer': '5', 'cortical_layer': '5'},
+            {'cre_type': 'ntsr1', 'target_layer': '6', 'cortical_layer': '6'},
+            {'cre_type': 'pvalb', 'target_layer': '6', 'cortical_layer': '6'},
+            {'cre_type': 'sst', 'target_layer': '6', 'cortical_layer': '6'},
+            {'cre_type': 'vip', 'target_layer': '6', 'cortical_layer': '6'},
         ]),
 
         ('Mouse Layer 2/3', [
-            {'cre_type': 'unknown', 'target_layer': '2/3'},
+            # {'cre_type': 'unknown', 'target_layer': '2/3'},
             #{'pyramidal': True, 'target_layer': '2/3'},
-            {'cre_type': 'pvalb', 'target_layer': '2/3'},
-            {'cre_type': 'sst', 'target_layer': '2/3'},
-            {'cre_type': 'vip', 'target_layer': '2/3'},
+            {'dendrite_type': 'spiny', 'target_layer': '2/3', 'cortical_layer': '2/3'},
+            {'cre_type': 'pvalb', 'target_layer': '2/3', 'cortical_layer': '2/3'},
+            {'cre_type': 'sst', 'target_layer': '2/3', 'cortical_layer': '2/3'},
+            {'cre_type': 'vip', 'target_layer': '2/3', 'cortical_layer': '2/3'},
         ]),
         
         ('Mouse Layer 4', [
-            {'cre_type': 'nr5a1', 'target_layer': '4'},
-            {'cre_type': 'pvalb', 'target_layer': '4'},
-            {'cre_type': 'sst', 'target_layer': '4'},
-            {'cre_type': 'vip', 'target_layer': '4'},
+            {'cre_type': 'nr5a1', 'target_layer': '4', 'cortical_layer': '4'},
```

Comment:
Why use both target_layer and cortical_layer?

Reply:
because sometimes you want to define by the target layer and other times by the annotated layer

Reply:
But I think this would make the cell class require _both_ to be true.

Reply:
I think I edited cell_class to make it an either or situation, but maybe we should look at it? When I tested it, the output seemed reasonable